### ISSUE TITLE
I don't know if the mod is supposed to work on forge, but it shows up as an option.

### ISSUE THREAD

**Issue by Drbruhtong** [Deadbones2000/Medieval-Origins#41]
If it is supposed to work, it doesn't. It will not appear on the mod list ingame, and the mod doesnt do anything when used in forge.

**Comment by Sweetroll-Queen** [comment]
same, I'm upset that I can't use it with the forge port of Origins even though it shows up as one for some reason :'<

**Comment by Moonively210** [comment]
Yes, I really hope this is fixed. I'm trying to make a fantasy RPG modpack and this is a crucial part of it.
